keps/2724-topology-aware-scheduling/README.md

```diff
@@ -666,6 +667,42 @@ for [Beta](#beta). The initial approach for the design is left in the
 [Support for ReplicatedJobs in JobSet](#support-for-replicatedjobs-in-jobset)
 section.
 
+#### Node failures
+
+Initially we plan to support node becoming not ready (as indicated in Node 
+`status.conditions.ready` field) and node being deleted. A new controller will 
+monitor nodes and will update each affected TAS workload with the information 
```

Comment:
Does this `TAS workload` mean running Workload? Or suspended Workload?
If this means running workload, does this indicate bypass the FIFO?

Reply:
It's for running Workloads, not suspended ones. Yes it would bypass the FIFO, and use the mechanism that is [in progress here](https://github.com/kubernetes-sigs/kueue/pull/5052)

Reply:
Thank you for clarifications. So, will the second pass of the scheduler does not execute preemption including reclaiming, right? If yes, could you expand "If no replacement is possible, the workload will be evicted." explanation so that we can clarify not to preempt and reclaim other running workloads.

Reply:
Yes, it wouldn't execute preemption, surely not in this version - might be if we gathered some users' feedback. 

>If yes, could you expand "If no replacement is possible, the workload will be evicted." explanation so that we can clarify not to preempt and reclaim other running workloads.

@pajakd Can you clarify that?

Reply:
Thank you. My concern here comment was preemptions. That was resolved.

> Yes, it wouldn't execute preemption, surely not in this version - might be if we gathered some users' feedback.

Basically, I think the preemption w/o requeue is very aggressive. So, if we want to introduce such a mechanism, we need to consider more. However, for now, we can postpone it to the future release.

Reply:
I agree, we may need to revisit preemptions. I guess we will learn more when we collect more feedback from users, and based on testing. For now this is an alpha feature, behind a dedicated feature gate. So, I think it is ok.